src/components/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import { TPAComponentProps } from '../../types';
+
+import WSRCalendar from 'wix-style-react/dist/src/Calendar';
+
+import { st, classes } from './DatePicker.st.css';
+import { DATA_HOOKS } from './constants';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+
+export type LanguageType =
+  | 'en'
+  | 'es'
+  | 'pt'
+  | 'fr'
+  | 'de'
+  | 'pl'
+  | 'it'
+  | 'ru'
+  | 'ja'
+  | 'ko'
+  | 'tr'
+  | 'sv'
+  | 'no'
+  | 'nl'
+  | 'da'
+  | 'zh'
+  | 'th'
+  | 'cs';
+
+export interface DatePickerProps extends TPAComponentProps {
+  /** The selected date */
+  value?: Date | string;
+  /** Callback function called with a Date or a Range whenever the user selects a day in the calendar */
+  onChange(selectedDays: string | Date): void;
+  /** DatePicker instance locale */
+  locale?: LanguageType | { distanceInWords?: {}; format?: {} };
```

Comment:
Does this allow to pass locales from outside?

Reply:
It should allow it but I haven't checked it yet.